content/docs/learning-guide/tutorials/entities-and-prefabs/entity-and-prefab-basics.md

```diff
@@ -0,0 +1,216 @@
+### Focus Mode and nested prefabs
+
+You can edit a nested prefab by opening the nested prefab in Focus Mode. In the following video, the nested prefab becomes the focus and a new entity is created within the context of the nested prefab.
+
+{{< video src="/images/learning-guide/tutorials/entities-and-prefabs/nested-focus-mode.mp4" info="Open a nested prefab for edit in focus mode." autoplay="true" loop="true" width="450" >}}
+
+## Detach a prefab instance
+
+When a prefab instance is detached, its link to the prefab file is broken and the prefab instance is converted to an entity. The hierarchy of the prefab is maintained by the new entity. To detach a prefab instance, **right-click** the prefab instance and select **Detach Prefab...** from the context menu, as demonstrated in the following video.
+
+{{< video src="/images/learning-guide/tutorials/entities-and-prefabs/detach-prefab.mp4" info="Detach a prefab instance." autoplay="true" loop="true" width="450" >}}
+
+{{< note >}}
+A prefab instance cannot be detached when it is in Focus Mode.
+{{< /note >}}
+
+## Duplicate an entity or a prefab instance
+
+Selected entities and prefab instances can be duplicated with either of the following methods:
+
+* **Right-click** a selected entity or prefab instance and choose **Duplicate** from the context menu.
+* With an entity or prefab instance selected, press **CTRL+D**.
+
+The duplicate appears below the source entity or prefab instance with a name that is identical to the source entity or prefab instance. Examining the duplicate in Entity Inspector shows the duplicate has a unique Entity ID.
+
+When multiple entities and prefabs are selected, they all are duplicated along with their hierarchies, as demonstrated in the following video.
+
+{{< video src="/images/learning-guide/tutorials/entities-and-prefabs/multiple-duplicates.mp4" info="Duplicating multiple entities and prefab instances." autoplay="true" loop="true" width="450" >}}
+
+In the following video, nested entities and prefab instances are duplicated in place within the hierarchy.
+
+{{< video src="/images/learning-guide/tutorials/entities-and-prefabs/nested-duplicate.mp4" info="Duplicating nested entities and prefab instances." autoplay="true" loop="true" width="450" >}}
+
+## Delete entities and prefab instances
+
+You can delete selected entities and prefab instances with either of the following methods:
+
+* **Right-click** a selected entity or prefab instance and choose **Delete** from the context menu.
+* With an entity or prefab instance selected, press **DEL**.
+
+{{< note >}}
+When an entity or a prefab instance is deleted from a level, you must save the level (press **CTRL+S**) to commit the change.
+
+When an entity or prefab instance is deleted from a prefab, you must [save the prefab](#saving-entities-and-prefabs) to commit the change.
+
+Deleting a prefab instance has no effect on the base prefab. The prefab instance is removed, but the `.prefab` file on disk is not changed. 
```

Comment:
Change to  '....has no effect on the prefab file used to created the prefab instance. The prefab instance is removed,...'

Reply:
done